<!DOCTYPE html> <html lang="en"> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Wadduwage Lab for Differentiable Microscopy (ùúïùúá)</title> <meta name="author" content=" "/> <meta name="description" content="In Wadduwage Lab, we work on novel computational microscopy solutions that can measure biological systems at their most information rich form, with minimum redundancy. "/> <meta name="keywords" content="wadduwagelab, differentiable microscopy"/> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light"/> <link rel="shortcut icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>‚öõÔ∏è</text></svg>"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://wadduwagelab.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="header-left"> <img src="/assets/img/logo.png" alt="logo.png" height="40px"> </div> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="//">Wadduwage Lab for Differentiable Microscopy (ùúïùúá)</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="year">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/wijethilake_et_al.png"></div> <div id="pub.1146056631" class="col-sm-8"> <div class="title">DEEP learning powered De-scattering with Excitation Patterning (DEEP) for high-throughput wide-field multiphoton microscopy</div> <div class="author"> Navodini Wijethilake,¬†Cheng Zheng,¬†Jong K. Park, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Murat Yildirim, Udith Haputhanthri, Peter T. C. So, Dushan N. Wadduwage' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p></p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/all-optical.gif"></div> <div id="pub.1146681163" class="col-sm-8"> <div class="title">Differentiable Microscopy Designs an All Optical Quantitative Phase Microscope</div> <div class="author"> Kithmini Herath,¬†Udith Haputhanthri,¬†Ramith Hettiarachchi, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Hasindu Kariyawasam, Azeem Ahmad, Balpreet S. Ahluwalia, Chamira U. S. Edussooriya, Dushan Wadduwage' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>arXiv</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2203.14944" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">arXiv</a> <a href="https://wadduwagelab.pages.dev" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">Website</a> </div> <div class="abstract hidden"> <p>Ever since the first microscope by Zacharias Janssen in the late 16th century, scientists have been inventing new types of microscopes for various tasks. Inventing a novel architecture demands years, if not decades, worth of scientific experience and creativity. In this work, we introduce Differentiable Microscopy (‚àÇŒº), a deep learning-based design paradigm, to aid scientists design new interpretable microscope architectures. Differentiable microscopy first models a common physics-based optical system however with trainable optical elements at key locations on the optical path. Using pre-acquired data, we then train the model end-to-end for a task of interest. The learnt design proposal can then be simplified by interpreting the learnt optical elements. As a first demonstration, based on the optical 4-f system, we present an all-optical quantitative phase microscope (QPM) design that requires no computational post-reconstruction. A follow-up literature survey suggested that the learnt architecture is similar to the generalized phase concept developed two decades ago. We then incorporate the generalized phase contrast concept to simplify the learning procedure. Furthermore, this physical optical setup is miniaturized using a diffractive deep neural network (D2NN). We outperform the existing benchmark for all-optical phase-to-intensity conversion on multiple datasets, and ours is the first demonstration of its kind on D2NNs. The proposed differentiable microscopy framework supplements the creative process of designing new optical systems and would perhaps lead to unconventional but better optical designs.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/haputhanthri_et_al.png"></div> <div id="pub.1146681164" class="col-sm-8"> <div class="title">Differentiable Microscopy for Content and Task Aware Compressive Fluorescence Imaging</div> <div class="author"> Udith Haputhanthri,¬†Andrew Seeber,¬†and¬†<em>Dushan Wadduwage</em> </div> <div class="periodical"> <em>arXiv</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The trade-off between throughput and image quality is an inherent challenge in microscopy. To improve throughput, compressive imaging under-samples image signals; the images are then computationally reconstructed by solving a regularized inverse problem. Compared to traditional regularizers, Deep Learning based methods have achieved greater success in compression and image quality. However, the information loss in the acquisition process sets the compression bounds. Further improvement in compression, without compromising the reconstruction quality is thus a challenge. In this work, we propose differentiable compressive fluorescence microscopy (‚àÇŒº) which includes a realistic generalizable forward model with learnable-physical parameters (e.g. illumination patterns), and a novel physics-inspired inverse model. The cascaded model is end-to-end differentiable and can learn optimal compressive sampling schemes through training data. With our model, we performed thousands of numerical experiments on various compressive microscope configurations. Our results suggest that learned sampling outperforms widely used traditional compressive sampling schemes at higher compressions (\times 100- 1000) in terms of reconstruction quality. We further utilize our framework for Task Aware Compression. The experimental results show superior performance on segmentation tasks even at extremely high compression (\times 4096).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/haputhanthri_et_al_2.png"></div> <div id="pub.1148150037" class="col-sm-8"> <div class="title">From Hours to Seconds: Towards 100x Faster Quantitative Phase Imaging via Differentiable Microscopy</div> <div class="author"> Udith Haputhanthri,¬†Kithmini Herath,¬†Ramith Hettiarachchi, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Hasindu Kariyawasam, Azeem Ahmad, Balpreet S. Ahluwalia, Chamira U. S. Edussooriya, Dushan N. Wadduwage' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>arXiv</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>With applications ranging from metabolomics to histopathology, quantitative phase microscopy (QPM) is a powerful label-free imaging modality. Despite significant advances in fast multiplexed imaging sensors and deep-learning-based inverse solvers, the throughput of QPM is currently limited by the speed of electronic hardware. Complementarily, to improve throughput further, here we propose to acquire images in a compressed form such that more information can be transferred beyond the existing electronic hardware bottleneck. To this end, we present a learnable optical compression-decompression framework that learns content-specific features. The proposed differentiable optical-electronic quantitative phase microscopy (‚àÇŒº) first uses learnable optical feature extractors as image compressors. The intensity representation produced by these networks is then captured by the imaging sensor. Finally, a reconstruction network running on electronic hardware decompresses the QPM images. The proposed system achieves compression of \times 64 while maintaining the SSIM of ‚àº0.90 and PSNR of ‚àº30 dB. The promising results demonstrated by our experiments open up a new pathway for achieving end-to-end optimized (i.e., optics and electronic) compact QPM systems that provide unprecedented throughput improvements.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1149039786" class="col-sm-8"> <div class="title">Differentiable Microscopy Designs an All Optical Quantitative Phase Microscope</div> <div class="author"> Kithmini Herath,¬†Udith Haputhanthri,¬†Ramith Hettiarachchi, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Hasindu Kariyawasam, Azeem Ahmad, Balpreet Ahluwalia, Chamira Edussooriya, Dushan Wadduwage' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Research Square</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Ever since the first microscope by Zacharias Janssen in the late 16th century, scientists have been inventing new types of microscopes for various tasks. Inventing a novel architecture demands years, if not decades, worth of scientific experience and creativity. In this work, we introduce Differentiable Microscopy (√¢ÀÜ‚Äö√é¬º), a deep learning-based design paradigm, to aid scientists design new interpretable microscope architectures. Differentiable microscopy first models a common physics-based optical system however with trainable optical elements at key locations on the optical path. Using pre-acquired data, we then train the model end-to-end for a task of interest. The learnt design proposal can then be simplified by interpreting the learnt optical elements. As a first demonstration, based on the optical 4-f system, we present an all-optical quantitative phase microscope (QPM) design that requires no computational post-reconstruction. A follow-up literature survey suggested that the learnt architecture is similar to the generalized phase concept developed two decades ago. We then incorporate the generalized phase contrast concept to simplify the learning procedure. Furthermore, we also demonstrate that similar results can be achieved by an uninterpretable learning based method, namely diffractive deep neural networks (D2NN). We outperform the existing benchmark for all-optical phase-to-intensity conversion on multiple datasets, and ours is the first demonstration of its kind on D2NNs. The proposed differentiable microscopy framework supplements the creative process of designing new optical systems and would perhaps lead to unconventional but better optical designs.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1149204268" class="col-sm-8"> <div class="title">Deep Optical Coding Design in Computational Imaging</div> <div class="author"> Henry Arguello,¬†Jorge Bacca,¬†Hasindu Kariyawasam, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Edwin Vargas, Miguel Marquez, Ramith Hettiarachchi, Hans Garcia, Kithmini Herath, Udith Haputhanthri, Balpreet Singh Ahluwalia, Peter So, Dushan N. Wadduwage, Chamira U. S. Edussooriya' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>arXiv</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Computational optical imaging (COI) systems leverage optical coding elements (CE) in their setups to encode a high-dimensional scene in a single or multiple snapshots and decode it by using computational algorithms. The performance of COI systems highly depends on the design of its main components: the CE pattern and the computational method used to perform a given task. Conventional approaches rely on random patterns or analytical designs to set the distribution of the CE. However, the available data and algorithm capabilities of deep neural networks (DNNs) have opened a new horizon in CE data-driven designs that jointly consider the optical encoder and computational decoder. Specifically, by modeling the COI measurements through a fully differentiable image formation model that considers the physics-based propagation of light and its interaction with the CEs, the parameters that define the CE and the computational decoder can be optimized in an end-to-end (E2E) manner. Moreover, by optimizing just CEs in the same framework, inference tasks can be performed from pure optics. This work surveys the recent advances on CE data-driven design and provides guidelines on how to parametrize different optical elements to include them in the E2E framework. Since the E2E framework can handle different inference applications by changing the loss function and the DNN, we present low-level tasks such as spectral imaging reconstruction or high-level tasks such as pose estimation with privacy preserving enhanced by using optimal task-based optical architectures. Finally, we illustrate classification and 3D object recognition applications performed at the speed of the light using all-optics DNN.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1149340127" class="col-sm-8"> <div class="title">Highly sensitive quantitative phase microscopy and deep learning complement whole genome sequencing for rapid detection of infection and antimicrobial resistance</div> <div class="author"> Azeem Ahmad,¬†Ramith Hettiarachchi,¬†Abdolrahman Khezri, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Balpreet Singh Ahluwalia, Dushan N. Wadduwage, Rafi Ahmad' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Abstract The current state-of-the-art infection and antimicrobial resistance diagnostics (AMR) is based mainly on culture-based methods with detection time of 48-96 hours. Slow diagnoses lead to adverse patient outcomes that directly correlate with the time taken to administer optimal antimicrobial. Mortality risk doubles with a 24-hour delay in providing appropriate antibiotics in cases of bacteremia. It is therefore essential to develop novel methods that can promptly and accurately diagnose microbial infections both at species and strain levels in clinical settings. Here, we demonstrate that the complimentary use of label-free optical assay with whole-genome sequencing (WGS) can enable high-speed culture-free diagnosis of AMR. Our assay is based on microscopy methods exploiting label free highly sensitive quantitative phase microscopy (QPM) followed by deep convolutional neural networks (DCNNs) based classification. We benchmarked our proposed workflow on twenty-one clinical isolates from four WHO priority pathogens that were antibiotic susceptibility testing (AST) phenotyped and their antimicrobial resistance (AMR) profile was determined by WGS. The proposed optical assay is in good agreement with the WGS characterization. Highly accurate classification based on the gram staining (100% for gramnegative and 83.4% for gram-positive), species (98.64), and wild-type/non-wild type (96.45%), as well as at the individual strain level (100% accurate in predicting 18 out of the 21 strains). These results demonstrate the potential of the QPM assay as a rapid and first-stage tool for species, resistance, and strain-level classification, which WGS can follow up for confirmation. Taken together, all this information is of high clinical importance. Such a workflow can facilitate efficient antimicrobial stewardship and prevent the spread of AMR.</p> </div> </div> </div> </li> </ol> <h2 class="year">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1133489272" class="col-sm-8"> <div class="title">Analysis of mutations in tumor and normal adjacent tissue via fluorescence detection</div> <div class="author"> Jennifer E. Kay,¬†Sheyla Mirabal,¬†William E. Briley, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Takafumi Kimoto, Theofilos Poutahidis, Timothy Ragan, Peter T. So, Dushan N. Wadduwage, Susan E. Erdman, Bevin P. Engelward' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Environmental and Molecular Mutagenesis</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Inflammation is a major risk factor for many types of cancer, including colorectal. There are two fundamentally different mechanisms by which inflammation can contribute to carcinogenesis. First, reactive oxygen and nitrogen species (RONS) can damage DNA to cause mutations that initiate cancer. Second, inflammatory cytokines and chemokines promote proliferation, migration, and invasion. Although it is known that inflammation-associated RONS can be mutagenic, the extent to which they induce mutations in intestinal stem cells has been little explored. Furthermore, it is now widely accepted that cancer is caused by successive rounds of clonal expansion with associated de novo mutations that further promote tumor development. As such, we aimed to understand the extent to which inflammation promotes clonal expansion in normal and tumor tissue. Using an engineered mouse model that is prone to cancer and within which mutant cells fluoresce, here we have explored the impact of inflammation on de novo mutagenesis and clonal expansion in normal and tumor tissue. While inflammation is strongly associated with susceptibility to cancer and a concomitant increase in the overall proportion of mutant cells in the tissue, we did not observe an increase in mutations in normal adjacent tissue. These results are consistent with opportunities for de novo mutations and clonal expansion during tumor growth, and they suggest protective mechanisms that suppress the risk of inflammation-induced accumulation of mutant cells in normal tissue.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1134598145" class="col-sm-8"> <div class="title">Excision of mutagenic replication-blocking lesions suppresses cancer but promotes cytotoxicity and lethality in nitrosamine-exposed mice</div> <div class="author"> Jennifer E. Kay,¬†Joshua J. Corrigan,¬†Amanda L. Armijo, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Ilana S. Nazari, Ishwar N. Kohale, Dorothea K. Torous, Svetlana L. Avlasevich, Robert G. Croy, Dushan N. Wadduwage, Sebastian E. Carrasco, Stephen D. Dertinger, Forest M. White, John M. Essigmann, Leona D. Samson, Bevin P. Engelward' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Summary N -nitrosodimethylamine (NDMA) is a DNA methylating agent that has been discovered to contaminate water, food and drugs. The alkyladenine glycosylase (AAG) removes methylated bases to initiate the base excision repair (BER) pathway. To understand how gene-environment interactions impact disease susceptibility, we studied Aag √¢ÀÜ‚Äô/√¢ÀÜ‚Äô and Aag -overexpressing mice that harbor increased levels of either replication-blocking lesions (3-methyladenine, or 3MeA) or strand breaks (BER intermediates), respectively. Remarkably, the disease outcome switched from cancer to lethality simply by changing AAG levels. To understand the underlying basis for this observation, we integrated a suite of molecular, cellular and physiological analyses. We found that unrepaired 3MeA is somewhat toxic but highly mutagenic (promoting cancer), whereas excess strand breaks are poorly mutagenic and highly toxic (suppressing cancer and promoting lethality). We demonstrate that the levels of a single DNA repair protein tips the balance between blocks and breaks, and thus dictates the disease consequences of DNA damage. </p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1136415780" class="col-sm-8"> <div class="title">Excision of mutagenic replication-blocking lesions suppresses cancer but promotes cytotoxicity and lethality in nitrosamine-exposed mice</div> <div class="author"> Jennifer E Kay,¬†Joshua J Corrigan,¬†Amanda L Armijo, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Ilana S Nazari, Ishwar N Kohale, Dorothea K Torous, Svetlana L Avlasevich, Robert G Croy, Dushan N Wadduwage, Sebastian E Carrasco, Stephen D Dertinger, Forest M White, John M Essigmann, Leona D Samson, Bevin P Engelward' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Cell Reports</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>N-Nitrosodimethylamine (NDMA) is a DNA-methylating agent that has been discovered to contaminate water, food, and drugs. The alkyladenine DNA glycosylase (AAG) removes methylated bases to initiate the base excision repair (BER) pathway. To understand how gene-environment interactions impact disease susceptibility, we study Aag-knockout (Aag<sup>-/-</sup>) and Aag-overexpressing mice that harbor increased levels of either replication-blocking lesions (3-methyladenine [3MeA]) or strand breaks (BER intermediates), respectively. Remarkably, the disease outcome switches from cancer to lethality simply by changing AAG levels. To understand the underlying basis for this observation, we integrate a suite of molecular, cellular, and physiological analyses. We find that unrepaired 3MeA is somewhat toxic, but highly mutagenic (promoting cancer), whereas excess strand breaks are poorly mutagenic and highly toxic (suppressing cancer and promoting lethality). We demonstrate that the levels of a single DNA repair protein tip the balance between blocks and breaks and thus dictate the disease consequences of DNA damage.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/zheng_et_al.png"></div> <div id="pub.1139474052" class="col-sm-8"> <div class="title">De-scattering with Excitation Patterning enables rapid wide-field imaging through scattering media</div> <div class="author"> Cheng Zheng,¬†Jong Kang Park,¬†Murat Yildirim, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Josiah R. Boivin, Yi Xue, Mriganka Sur, Peter T. C. So, Dushan N. Wadduwage' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Science Advances</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Nonlinear optical microscopy has enabled in vivo deep tissue imaging on the millimeter scale. A key unmet challenge is its limited throughput especially compared to rapid wide-field modalities that are used ubiquitously in thin specimens. Wide-field imaging methods in tissue specimens have found successes in optically cleared tissues and at shallower depths, but the scattering of emission photons in thick turbid samples severely degrades image quality at the camera. To address this challenge, we introduce a novel technique called De-scattering with Excitation Patterning or "DEEP," which uses patterned nonlinear excitation followed by computational imaging-assisted wide-field detection. Multiphoton temporal focusing allows high-resolution excitation patterns to be projected deep inside specimen at multiple scattering lengths due to the use of long wavelength light. Computational reconstruction allows high-resolution structural features to be reconstructed from tens to hundreds of DEEP images instead of millions of point-scanning measurements.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1143084694" class="col-sm-8"> <div class="title">De-scattering with Excitation Patterning (DEEP) in temporal focusing microscopy</div> <div class="author"> Cheng Zheng,¬†Jong Kang Park,¬†Murat Yildirim, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Josiah R. Boivin, Yi Xue, Mriganka Sur, Peter T. C. So, Dushan N. Wadduwage' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The scattering in turbid samples severely degrades image quality of temporal focusing microscopy. To address this challenge, we introduce a novel technique called Descattering with Excitation Patterning, which uses patterned excitation followed by wide-field detection.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1143590908" class="col-sm-8"> <div class="title">Physics Augmented U-Net: A High-Frequency Aware Generative Prior for Microscopy</div> <div class="author"> Jathurshan Pradeepkumar,¬†Mithunjha Anandakumar,¬†Vinith Kugathasan, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Andrew Seeber, Dushan N. Wadduwage' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Abstract A key challenge in optical microscopy is to image fast at high-resolution. To address this problem, we propose √¢‚Ç¨≈ìPhysics Augmented U-Net√¢‚Ç¨¬ù, which combines deep learning and structured illumination microscopy (SIM). In SIM, the structured illumination aliases out-of-band high-frequencies to the passband of the microscope; thus SIM captures some high-frequencies even when the image is sampled at low-resolution. To utilize these features, we propose a three-element method: 1) a modified U-Net model, 2) a physics-based forward model of SIM 3) an inference algorithm combining the two models. The modified U-Net architecture is similar to the seminal work, but the bottleneck is modified by concatenating two latent vectors, one encoding low-frequencies (LFLV), and the other encoding high-frequencies (HFLV). LFLV is learned by U-Net contracting path, and HFLV is learned by a second encoding path. In the inference mode, the high-frequency encoder is removed; HFLV is then optimized to fit the measured microscopy images to the output of the forward model for the generated image by the U-Net. We validated our method on two different datasets under different experimental conditions. Since a latent vector is optimized instead of a 2D image, the inference mode is less computationally complex. The proposed model is also more stable compared to other generative prior-based methods. Finally, as the forward model is independent of the U-Net, Physics Augmented U-Net can enhance resolution on any variation of SIM without further retraining.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/wadduwage_et_al.png"></div> <div id="pub.1143692893" class="col-sm-8"> <div class="title">Differentiable microscopy (‚àÇŒº) for high-throughput imaging cytometry</div> <div class="author"> Dushan N. Wadduwage,¬†and¬†Udith Haputhanthri</div> <div class="periodical"> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Imaging cytometry is a vital tool in many bio-imaging studies. Here we introduce, Differentiable Microscopy () a general end-to-end optimizable design framework to improve the throughput of imaging cytometry using content aware sampling and reconstruction.</p> </div> </div> </div> </li> </ol> <h2 class="year">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1125514112" class="col-sm-8"> <div class="title">High-throughput three-dimensional imaging cytometer for subnuclear foci quantification (Conference Presentation)</div> <div class="author"> Cheng Zheng,¬†Dushan N. Wadduwage,¬†Jong Park, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Christy Chao, Jenny Kay, Norah Owiti, Zachary Nagel, Bevin P. Engelward, Peter T. C. So' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p></p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1129936835" class="col-sm-8"> <div class="title">Advances in Chromatin and Chromosome Research: Perspectives from Multiple Fields</div> <div class="author"> Andrews Akwasi Agbleke,¬†Assaf Amitai,¬†Jason D. Buenrostro, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Aditi Chakrabarti, Lingluo Chu, Anders S. Hansen, Kristen M. Koenig, Ajay S. Labade, Sirui Liu, Tadasu Nozaki, Sergey Ovchinnikov, Andrew Seeber, Haitham A. Shaban, Jan-Hendrik Spille, Andrew D. Stephens, Jun-Han Su, Dushan Wadduwage' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>Molecular Cell</em> 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Nucleosomes package genomic DNA into chromatin. By regulating DNA access for transcription, replication, DNA repair, and epigenetic modification, chromatin forms the nexus of most nuclear processes. In addition, dynamic organization of chromatin underlies both regulation of gene expression and evolution of chromosomes into individualized sister objects, which can segregate cleanly to different daughter cells at anaphase. This collaborative review shines a spotlight on technologies that will be crucial to interrogate key questions in chromatin and chromosome biology including state-of-the-art microscopy techniques, tools to physically manipulate chromatin, single-cell methods to measure chromatin accessibility, computational imaging with neural networks and analytical tools to interpret chromatin structure and dynamics. In addition, this review provides perspectives on how these tools can be applied to specific research fields such as genome stability and developmental biology and to test concepts such as phase separation of chromatin.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1132525325" class="col-sm-8"> <div class="title">Excision of Mutagenic Replication-Blocking Lesions Suppresses Cancer But Promotes Cytotoxicity and Lethality in Nitrosamine-Exposed Mice</div> <div class="author"> Jennifer Elizabeth Kay,¬†Joshua J. Corrigan,¬†Amanda L. Armijo, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Ilana S. Nazari, Ishwar N. Kohale, Dorothea K. Torous, Svetlana L. Avlasevich, Robert G. Croy, Dushan N. Wadduwage, Sebastian E. Carrasco, Stephen D. Dertinger, Forest M. White, John M. Essigmann, Leona D. Samson, Bevin P. Engelward' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>SSRN Electronic Journal</em> 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p></p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1137982689" class="col-sm-8"> <div class="title">Signal to Noise Considerations in Single Pixel Multiphoton Microscopy</div> <div class="author"> <em>Dushan Wadduwage</em> </div> <div class="periodical"> <em>iScience Notes</em> 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Single-pixel imaging geometries for wide-field multiphoton microscopy (SPx-MPM) have emerged as a contender to conventional point-scanning multiphoton systems (PS-MPM) for deep tissue imaging. These systems are thought to be faster due to their multiplexed imaging capabilities with higher photon throughput. In this study we numerically compare the signal to noise metrics of the SPx-MPM to the PS-MPM systems. Our results suggest that PS-MPM systems outperform SPx-MPM systems, despite their higher photon throughput.</p> </div> </div> </div> </li> </ol> <h2 class="year">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"><img class="preview z-depth-1 rounded" src="/assets/img/publication_preview/coucheron_et_al.png"></div> <div id="pub.1114012042" class="col-sm-8"> <div class="title">Chip-Based Resonance Raman Spectroscopy Using Tantalum Pentoxide Waveguides</div> <div class="author"> David A. Coucheron,¬†Dushan N. Wadduwage,¬†G. Senthil Murugan, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Peter T. C. So, Balpreet S. Ahluwalia' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>IEEE Photonics Technology Letters</em> 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Blood analysis is an important diagnostic tool, as it provides a wealth of information about the patient√¢‚Ç¨‚Ñ¢s health. Raman spectroscopy is a promising tool for blood analysis, but widespread clinical application is limited by its low signal strength, as well as complex and costly instrumentation. The growing field of waveguide-based Raman spectroscopy tries to solve these challenges by working toward fully integrated Raman sensors with increased interaction areas. In this letter, we demonstrate resonance Raman measurements of hemoglobin, a crucial component of blood, at 532-nm excitation using a tantalum pentoxide (Ta2O5) waveguide platform. We have also characterized the background signal from Ta2O5 waveguide material when excited at 532 nm. In addition, we demonstrate spontaneous Raman measurements of isopropanol and methanol using the same platform. Our results suggest that Ta2O5 is a promising waveguide platform for resonance Raman spectroscopy at 532 nm and, in particular, for blood analysis.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1118852525" class="col-sm-8"> <div class="title">De-scattering with Excitation Patterning (DEEP) Enables Rapid Wide-field Imaging Through Scattering Media</div> <div class="author"> Dushan N. Wadduwage,¬†Jong Kang Park,¬†Josiah R. Boivin, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Yi Xue, Peter T. C. So' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>arXiv</em> 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>From multi-photon imaging penetrating millimeters deep through scattering biological tissue, to super-resolution imaging conquering the diffraction limit, optical imaging techniques have greatly advanced in recent years. Notwithstanding, a key unmet challenge in all these imaging techniques is to perform rapid wide-field imaging through a turbid medium. Strategies such as active wave-front correction and multi-photon excitation, both used for deep tissue imaging; or wide-field total-internal-refection illumination, used for super-resolution imaging; can generate arbitrary excitation patterns over a large field-of-view through or under turbid media. In these cases, throughput advantage gained by wide-field excitation is lost due to the use of point detection. To address this challenge, here we introduce a novel technique called De-scattering with Excitation Patterning, or ‚ÄôDEEP‚Äô, which uses patterned excitation followed by wide-field detection with computational imaging. We use two-photon temporal focusing (TFM) to demonstrate our approach at multiple scattering lengths deep in tissue. Our results suggest that millions of point-scanning measurements could be substituted with tens to hundreds of DEEP measurements with no compromise in image quality.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1118873499" class="col-sm-8"> <div class="title">Multiline Orthogonal Scanning Temporal Focusing (mosTF) Microscopy for Scattering Reduction in High-speed in vivo Brain Imaging</div> <div class="author"> Yi Xue,¬†Josiah R. Boivin,¬†Dushan N. Wadduwage, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jong Kang Park, Elly Nedivi, Peter T. C. So' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>arXiv</em> 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Temporal focusing two-photon microscopy enables high resolution imaging of fine structures in vivo over a large volume. A limitation of temporal focusing is that signal-to-background ratio and resolution degrade rapidly with increasing imaging depth. This degradation originates from the scattered emission photons are widely distributed resulting in a strong background. We have developed Multiline Orthogonal Scanning Temporal Focusing (mosTF) microscopy that overcomes this problem. mosTF captures a sequence of images at each scan location of the excitation line, followed by a reconstruction algorithm reassigns scattered photons back to the correct scan position. We demonstrate mosTF by acquiring mice neuronal images in vivo. Our results show remarkably improvements with mosTF for in vivo brain imaging while maintaining its speed advantage.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1123793778" class="col-sm-8"> <div class="title">3D Deep Learning Enables Fast Imaging of Spines through Scattering Media by Temporal Focusing Microscopy</div> <div class="author"> Zhun Wei,¬†Josiah R. Boivin,¬†Yi Xue, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Xudong Chen, Peter T. C. So, Elly Nedivi, Dushan N. Wadduwage' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>arXiv</em> 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Today the gold standard for in vivo imaging through scattering tissue is the point-scanning two-photon microscope (PSTPM). Especially in neuroscience, PSTPM is widely used for deep-tissue imaging in the brain. However, due to sequential scanning, PSTPM is slow. Temporal focusing microscopy (TFM), on the other hand, focuses femtosecond pulsed laser light temporally, while keeping wide-field illumination, and is consequently much faster. However, due to the use of a camera detector, TFM suffers from the scattering of emission photons. As a result, TFM produces images of poor spatial resolution and signal-to-noise ratio (SNR), burying fluorescent signals from small structures such as dendritic spines. In this work, we present a data-driven deep learning approach to improve resolution and SNR of TFM images. Using a 3D convolutional neural network (CNN) we build a map from TFM to PSTPM modalities, to enable fast TFM imaging while maintaining high-resolution through scattering media. We demonstrate this approach for in vivo imaging of dendritic spines on pyramidal neurons in the mouse visual cortex. We show that our trained network rapidly outputs high-resolution images that recover biologically relevant features previously buried in the scattered fluorescence in the TFM images. In vivo imaging that combines TFM and the proposed 3D convolution neural network is one to two orders of magnitude faster than PSTPM but retains the high resolution and SNR necessary to analyze small fluorescent structures. The proposed 3D convolution deep network could also be potentially beneficial for improving the performance of many speed-demanding deep-tissue imaging applications such as in vivo voltage imaging.</p> </div> </div> </div> </li> </ol> <h2 class="year">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1101827223" class="col-sm-8"> <div class="title">Patterned Illumination Wide-Field Two-Photon Microscopy for Enhanced Axial Resolution and Deep Imaging</div> <div class="author"> <em>Dushan Wadduwage</em>,¬†Jong K. Park,¬†and¬†Peter T. So</div> <div class="periodical"> 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We demonstrate a simple and programmable implementation of pattern scanning temporal focusing technique, by employing a digital micromirror device, for enhancing axial resolution.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1106070383" class="col-sm-8"> <div class="title">Automated fluorescence intensity and gradient analysis enables detection of rare fluorescent mutant cells deep within the tissue of RaDR mice</div> <div class="author"> Dushan N. Wadduwage,¬†Jennifer Kay,¬†Vijay Raj Singh, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Orsolya Kiraly, Michelle R. Sukup-Jackson, Jagath Rajapakse, Bevin P. Engelward, Peter T. C. So' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Scientific Reports</em> 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Homologous recombination (HR) events are key drivers of cancer-promoting mutations, and the ability to visualize these events in situ provides important information regarding mutant cell type, location, and clonal expansion. We have previously created the Rosa26Direct Repeat (RaDR) mouse model wherein HR at an integrated substrate gives rise to a fluorescent cell. To fully leverage this in√Ç situ approach, we need better ways to quantify rare fluorescent cells deep within tissues. Here, we present a robust, automated event quantification algorithm that uses image intensity and gradient features to detect fluorescent cells in deep tissue specimens. To analyze the performance of our algorithm, we simulate fluorescence behavior in tissue using Monte Carlo methods. Importantly, this approach reduces the potential for bias in manual counting and enables quantification of samples with highly dense HR events. Using this approach, we measured the relative frequency of HR within a chromosome and between chromosomes and found that HR within a chromosome is more frequent, which is consistent with the close proximity of sister chromatids. Our approach is both objective and highly rapid, providing a powerful tool, not only to researchers interested in HR, but also to many other researchers who are similarly using fluorescence as a marker for understanding mammalian biology in tissues.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1107777592" class="col-sm-8"> <div class="title">Scattering reduction by structured light illumination in line-scanning temporal focusing microscopy.</div> <div class="author"> Yi Xue,¬†Kalen P Berry,¬†Josiah R Boivin, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Dushan Wadduwage, Elly Nedivi, Peter T C So' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Biomedical optics express</em> 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Line-scanning temporal focusing microscopy (LineTFM) is capable of imaging biological samples more than 10 times faster than two-photon laser point-scanning microscopy (TPLSM), while achieving nearly the same lateral and axial spatial resolution. However, the image contrast taken by LineTFM is lower than that by TPLSM because LineTFM is severely influenced by biological tissue scattering. To reject the scattered photons, we implemented LineTFM using both structured illumination and uniform illumination combined with the HiLo post-processing algorithm, called HiLL microscopy (HiLo-Line-scanning temporal focusing microscopy). HiLL microscopy significantly reduces tissue scattering and improves image contrast. We demonstrate HiLL microscopy with <i>in vivo</i> brain imaging. This approach could potentially find applications in monitoring fast dynamic events and in mapping high resolution structures over a large volume.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1118870932" class="col-sm-8"> <div class="title">Chip-based Resonance Raman Spectroscopy Using Tantalum Pentoxide Waveguides</div> <div class="author"> David A. Coucheron,¬†Dushan N. Wadduwage,¬†G. Senthil Murugan, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Peter T. C. So, Balpreet S. Ahluwalia' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>arXiv</em> 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Blood analysis is an important diagnostic tool, as it provides a wealth of information about the patient health. Raman spectroscopy is a promising tool in blood analysis, but widespread clinical application is limited by its low signal strength, as well as complex and costly instrumentation. The growing field of waveguide-based Raman spectroscopy tries to solve these challenges by working towards fully integrated Raman sensors with increased interaction areas. In this work, we demonstrate on-chip resonance Raman measurements of hemoglobin, a crucial component of blood, at 532 nm excitation using a tantalum pentoxide (Ta2O5) waveguide platform. We have also characterized the background signal from Ta2O5 waveguide material when excited at 532 nm. In addition, we demonstrate spontaneous Raman measurements of iso-propanol and methanol using the same platform. Our results suggest that Ta2O5 is a promising waveguide platform for resonance Raman spectroscopy at 532 nm, and in particular for blood analysis.</p> </div> </div> </div> </li> </ol> <h2 class="year">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1085525399" class="col-sm-8"> <div class="title">Near-common-path interferometer for imaging Fourier-transform spectroscopy in wide-field microscopy.</div> <div class="author"> Dushan N Wadduwage,¬†Vijay Raj Singh,¬†Heejin Choi, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Zahid Yaqoob, Hans Heemskerk, Paul Matsudaira, Peter T C So' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Optica</em> 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Imaging Fourier-transform spectroscopy (IFTS) is a powerful method for biological hyperspectral analysis based on various imaging modalities, such as fluorescence or Raman. Since the measurements are taken in the Fourier space of the spectrum, it can also take advantage of compressed sensing strategies. IFTS has been readily implemented in high-throughput, high-content microscope systems based on wide-field imaging modalities. However, there are limitations in existing wide-field IFTS designs. Non-common-path approaches are less phase-stable. Alternatively, designs based on the common-path Sagnac interferometer are stable, but incompatible with high-throughput imaging. They require exhaustive sequential scanning over large interferometric path delays, making compressive strategic data acquisition impossible. In this paper, we present a novel phase-stable, near-common-path interferometer enabling high-throughput hyperspectral imaging based on strategic data acquisition. Our results suggest that this approach can improve throughput over those of many other wide-field spectral techniques by more than an order of magnitude without compromising phase stability.</p> </div> </div> </div> </li></ol> <h2 class="year">2016</h2> <ol class="bibliography"></ol> <h2 class="year">2015</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1038270626" class="col-sm-8"> <div class="title">Three√¢‚Ç¨¬êdimensional image cytometer based on widefield structured light microscopy and high√¢‚Ç¨¬êspeed remote depth scanning</div> <div class="author"> Heejin Choi,¬†Dushan N. Wadduwage,¬†Ting Yuan Tu, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Paul Matsudaira, Peter T. C. So' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Cytometry Part A</em> 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>A high throughput 3D image cytometer have been developed that improves imaging speed by an order of magnitude over current technologies. This imaging speed improvement was realized by combining several key components. First, a depth-resolved image can be rapidly generated using a structured light reconstruction algorithm that requires only two wide field images, one with uniform illumination and the other with structured illumination. Second, depth scanning is implemented using the high speed remote depth scanning. Finally, the large field of view, high NA objective lens and the high pixelation, high frame rate sCMOS camera enable high resolution, high sensitivity imaging of a large cell population. This system can image at 800 cell/sec in 3D at submicron resolution corresponding to imaging 1 million cells in 20 min. The statistical accuracy of this instrument is verified by quantitatively measuring rare cell populations with ratio ranging from 1:1 to 1:10(5) . √Ç¬© 2014 International Society for Advancement of Cytometry. </p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1046365587" class="col-sm-8"> <div class="title">Subnuclear foci quantification using high-throughput 3D image cytometry</div> <div class="author"> Dushan N. Wadduwage,¬†Marcus Parrish,¬†Heejin Choi, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Bevin P. Engelward, Paul Matsudaira, Peter T. C. So' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p></p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1098903283" class="col-sm-8"> <div class="title">Subnuclear foci quantification using high-throughput 3D image cytometry</div> <div class="author"> Dushan N. Wadduwage,¬†Marcus Parrish,¬†Heejin Choi, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Bevin P. Engelward, Paul Matsudaira, Peter T. C. So' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p></p> </div> </div> </div> </li> </ol> <h2 class="year">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">test</abbr></div> <div id="pub.1009664368" class="col-sm-8"> <div class="title">Rosa26-GFP Direct Repeat (RaDR-GFP) Mice Reveal Tissue- and Age-Dependence of Homologous Recombination in Mammals In Vivo</div> <div class="author"> Michelle R. Sukup-Jackson,¬†Orsolya Kiraly,¬†Jennifer E. Kay, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Li Na, Elizabeth A. Rowland, Kelly E. Winther, Danielle N. Chow, Takafumi Kimoto, Tetsuya Matsuguchi, Vidya S. Jonnalagadda, Vilena I. Maklakova, Vijay R. Singh, Dushan N. Wadduwage, Jagath Rajapakse, Peter T. C. So, Lara S. Collier, Bevin P. Engelward' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>PLoS Genetics</em> 2014 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Homologous recombination (HR) is critical for the repair of double strand breaks and broken replication forks. Although HR is mostly error free, inherent or environmental conditions that either suppress or induce HR cause genomic instability. Despite its importance in carcinogenesis, due to limitations in our ability to detect HR in vivo, little is known about HR in mammalian tissues. Here, we describe a mouse model in which a direct repeat HR substrate is targeted to the ubiquitously expressed Rosa26 locus. In the Rosa26 Direct Repeat-GFP (RaDR-GFP) mice, HR between two truncated EGFP expression cassettes can yield a fluorescent signal. In-house image analysis software provides a rapid method for quantifying recombination events within intact tissues, and the frequency of recombinant cells can be evaluated by flow cytometry. A comparison among 11 tissues shows that the frequency of recombinant cells varies by more than two orders of magnitude among tissues, wherein HR in the brain is the lowest. Additionally, de novo recombination events accumulate with age in the colon, showing that this mouse model can be used to study the impact of chronic exposures on genomic stability. Exposure to N-methyl-N-nitrosourea, an alkylating agent similar to the cancer chemotherapeutic temozolomide, shows that the colon, liver and pancreas are susceptible to DNA damage-induced HR. Finally, histological analysis of the underlying cell types reveals that pancreatic acinar cells and liver hepatocytes undergo HR and also that HR can be specifically detected in colonic somatic stem cells. Taken together, the RaDR-GFP mouse model provides new understanding of how tissue and age impact susceptibility to HR, and enables future studies of genetic, environmental and physiological factors that modulate HR in mammals. </p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="pub.1021550429" class="col-sm-8"> <div class="title">Depth resolved hyperspectral imaging spectrometer based on structured light illumination and Fourier transform interferometry</div> <div class="author"> Heejin Choi,¬†<em>Dushan Wadduwage</em>,¬†Paul T. Matsudaira, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Peter T.C. So' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Biomedical Optics Express</em> 2014 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>A depth resolved hyperspectral imaging spectrometer can provide depth resolved imaging both in the spatial and the spectral domain. Images acquired through a standard imaging Fourier transform spectrometer do not have the depth-resolution. By post processing the spectral cubes (x, y, √é¬ª) obtained through a Sagnac interferometer under uniform illumination and structured illumination, spectrally resolved images with depth resolution can be recovered using structured light illumination algorithms such as the HiLo method. The proposed scheme is validated with in vitro specimens including fluorescent solution and fluorescent beads with known spectra. The system is further demonstrated in quantifying spectra from 3D resolved features in biological specimens. The system has demonstrated depth resolution of 1.8 √é¬ºm and spectral resolution of 7 nm respectively. </p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> ¬© Copyright 2023 . </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </body> </html>